saleor/graphql/schema.graphql

```diff
@@ -4356,7 +4357,8 @@ type OrderLine implements Node {
   id: ID!
   productName: String!
   variantName: String!
-  productSku: String!
+  productSku: String
+  productVariantId: String
```

Comment:
Is this needed for frontend? Where does this change come from?

Reply:
It came from picking changes in this file from master. Interesting it shows up in git changes record, but not in github diff.